config

```diff
@@ -21,6 +21,7 @@ INTERFACE=example_vpn
 
 # CIDR wireguard interface address.
```

Comment:
Use undefined vars to represent unset values. This requires more backend code to handle these cases but makes it easier for the user to to consume.

```
# Wireguard interface name. (leave blank to disable).
INTERFACE_ADDR=172.31.255.10/32
INTERFACE_ADDR_6=

# Allowed IP's (CIDR) on wireguard; for boot this should be the peer (server) (leave blank to disable).
ALLOWED_IPS=172.31.255.254/32
ALLOWED_IPS_6=

# Persist wireguard interface after initramfs exits (leave blank to disable).
PERSISTENT=Y
```